app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/components/UserAcronymCircle.kt

```diff
@@ -0,0 +1,55 @@
+package com.realityexpander.tasky.agenda_feature.presentation.common
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import com.realityexpander.tasky.agenda_feature.presentation.common.util.getUserAcronym
+import com.realityexpander.tasky.core.presentation.common.modifiers.DP
+import com.realityexpander.tasky.core.presentation.theme.TaskyLightBlue
+
+@Composable
+fun UserAcronymCircle(
+    modifier: Modifier = Modifier,
+    username: String? = "??",
```

Comment:
If you allow nullable values, why not then set it to null initially if the username is non-existent?

Reply:
Have it be null makes the composable jump around on the first recompose view and is visually jarring. I will work on making it stable and remove the "??"